pom.xml

```diff
@@ -26,5 +45,13 @@
           <version>4.7</version>
           <scope>test</scope>
       </dependency>
+      <dependency>
```

Comment:
It seems that for unit testing they're using a different version. Will have to look further into it, though.